### ISSUE TITLE
Replays Freezing Bug

### ISSUE THREAD

**Issue by Seimix** [Robosturm/Commander_Wars#392]
**Describe the bug**
When messing with the slider in Replays or right-clicking, it freezes the entire game.

**To Reproduce**
Simply play a replay, and right-click on the map or mess with the slider / "speed" button in it to cause it to freeze. 

**Additional Context**
I'm using a replay of a Multiplayer game btw.

**Comment by Robosturm** [comment]
Can you please take a look if you have crash log file for the freeze?
I think i have fixed that issue already but a log would help to verify that.
Thx

**Comment by Seimix** [comment]
No... I don't have a crash log for it.

Let me rephrase this: 
When you mess with the slider or try skipping days in a replay, the replay gets stuck in a kind of pause-like state. And clicking the speed-up button doesn't do anything except "stop" the replay.
Example: https://www.youtube.com/watch?v=fgVE4ai6gRU&feature=youtu.be&ab_channel=DerpyKong

Some other notes:
- The mods I'm using are: AW:DS Weather, Transfer Units, Sell Units, Debuff Perks, AW-Styled Icons, AW2 Image mod, and Xanari Crew
- Random weather IS on
- Match was done on a custom map in multiplayer

**Comment by Robosturm** [comment]
Okay i thought the game freezes.
In that case i fixed it with the commit.
Thx for clarifying that.
